corona-warn-app/cwa-app-android
# [Fix] Correct destination for warn others notification (EXPOSUREAPP-12203)

### Whats new
Correct destination for warn others notification: now leading to submissionResultPositiveOtherWarningNoConsentFragment.

### Jira
[Ticket](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12203)

### How to test
1) scan positive test
2) proceed with submission flow
3) revoke consent for warn others
4) press next and on pop-up window press "Do not warn others"
5) check the main screen: there should be Positive test result with "Warn others" button
6) time travel to 2+ hours (I used 2h and some extra minutes)
7) notification "You can help" should appear 
8) tap on notification should lead you to Warn others fragment (not to "test result is ready" with missing consent)
